internal/driver/glfw/window_wasm.go

```diff
@@ -552,6 +531,7 @@ func (w *window) create() {
 	for _, fn := range w.pending {
 		fn()
 	}
+	w.pending = nil
```

Comment:
Can you comment on this line?

Reply:
I just added in the same as what the general desktop driver does on the corresponding line https://github.com/fyne-io/fyne/blob/66c8de3d389aa7d33c72ce371e93c09c397ad8ba/internal/driver/glfw/window_desktop.go#L787. I can refactor it out to a shared function between the two implementations if you'd like?

Reply:
I extracted into a separate function and also cleaned up a few other minor things I saw

Reply:
I have fixed the conflict. Would you kindly re-review? @andydotxyz

Reply:
Thanks for this, sorry for the delay